startbootstrap-sb-admin-2-gh-pages/server.js

```diff
@@ -12,10 +12,8 @@ app.engine("html", require("ejs").renderFile);
 app.use(express.static("./"));
 app.use(require("connect-livereload")());
 
-app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
-
-
+app.use(bodyParser.json())
```

Comment:
The reason I used app.use(express.json()) was because I thought express.json replaced bodyparser in the latest versions. Either is fine I think though!

Reply:
I think you are right actually but I think both are fine.